t3/main.py

```diff
@@ -14,7 +14,6 @@
 
```

Comment:
The commit message says "Update rmg_simulator.py", the intention is probably "main.py"